portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -58,4 +64,43 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(eventEntity);
     response.sendRedirect("/testForm.html");
   }
+  //TODO add doGet method
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Event");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+        for (Entity entity : results.asIterable()){
+            String organizer = (String) entity.getProperty("organizer");
+            String eventName = (String) entity.getProperty("eventName");
+            String location = (String) entity.getProperty("location");
+            String description = (String) entity.getProperty("description");
+            Date date = (Date) entity.getProperty("date");
```

Comment:
There's a CustomDate class in this project too: https://github.com/jamilao/sps-team-project/blob/master/portfolio/src/main/java/com/google/sps/servlets/CustomDate.java

Is there any reason to use CustomDate instead of Date? I'm good with Date, just double checking